cx_Freeze/main.py

```diff
@@ -157,12 +157,13 @@ def ParseCommandLine():
 def main():
     options = ParseCommandLine()
     executables = [cx_Freeze.Executable(options.script,
-                                        initScript = options.initScript,
-                                        base = options.baseName,
-                                        icon = options.icon,
-                                        targetName = options.targetName)
+                                        options.initScript,
+                                        options.baseName,
+                                        options.targetName,
+                                        options.icon)
```

Comment:
Why these changes? They don't have anything to do with the bug, do they? Can you remove them?

Reply:
Is there some special reason for not merging this commit? I have problem of fetching from pypi in the CI/CD environment. locally I patched the same file and it works. The problem is that on PyPi I don't see package updated and there is no development version available in pypi either so I can't rely on CI/CD to buld my executable for some projects. 
At least in my project nothing is broken by removing kwargs and making them args.

Reply:
Can you explain why switching from keyword args to positional args is an improvement? It may work just fine, but what is the advantage to making the change? If you can explain that and it makes sense to me I'll be happy to merge that commit!

Reply:
Well I am certain that having a keyword arguments is nicer as you do not need to pay attention that much on names of parameters and their order. 
But however you had a grave bug in important project. You could either push update faster or accept marcelo commit.

Its sort of a big problem when Pypi start hosting projects as packages that fail. There should be checking for that automated for every working supported version of python. Imagine when I needed to edit a python package in venv to have it work with 3.8 and even then the CI/CD couldn't do that so I needed to downgrade.

I mean project is nice but its not good to leave people downloading your package from pypi that do not work for over a full month.